llama_toolchain/cli/stack/run.py

```diff
@@ -60,14 +59,22 @@ def _add_arguments(self):
             help="Disable IPv6 support",
             default=False,
         )
+        self.parser.add_argument(
```

Comment:
here as well, and especially important here. we should only take config as an argument and make all necessary changes to make that work. and the argument should be positional not optional. i.e., it should be `'config'` and not `'--config'`